website/src/types/wastewaterConfig.ts

```diff
@@ -10,6 +10,8 @@ export const wastewaterConfig = {
     browseDataUrl: 'https://wise-loculus.genspectrum.org',
     browseDataDescription: 'Browse the data in the WISE Loculus instance.',
     lapisBaseUrl: 'https://api.wise-loculus.genspectrum.org',
+    wasapLapisBaseUrl: 'https://lapis.wasap.genspectrum.org/',
+    covSpectrumLapisBaseUrl: 'https://lapis.cov-spectrum.org/open/v2',
```

Comment:
Now we still have it twice in the code (the other one is here: https://github.com/GenSpectrum/dashboards/blob/cff6db89347108d0f98c2fd292a42d9cb5dfa818/backend/src/main/resources/application-dashboards-prod.yaml#L5). 

But I guess that's ok for now.